changelog/4327.bugfix.rst

```diff
@@ -0,0 +1 @@
+Loosen the definition of what ``approx`` considers a sequence
```

Comment:
If we have decided this, please specify, I suggest:

```
``approx`` now works with more generic containers, more precisely instances of ``Iterable`` instead of more restrict ``Sequence``.
```

Reply:
I went for the slightly different

> ``approx`` again works with more generic containers, more precisely instances of ``Iterable`` and ``Sized`` instead of more restrictive ``Sequence``.

To emphasises the actual restriction and that it's restoring a previous compatibility